### PR TITLE
feat: add worldname to the notify flags

### LINKED ISSUE #3456: Add Worldname to notify flags
### What feature do you want to see added?

Add the world name to the notify flags to distinguish them on multi world servers.

### Are there any alternatives?

nope

### Anything else?

Already implemented. Creating pull request as soon as possible after this

### PR DESCRIPTION
**Pull request by Xaver106**
## Overview

Fixes #3456 

## Description

Adds the world name to the notify flag messages. It will now look like this:
image

## Submitter Checklist

- [X] Make sure you are opening from a topic branch (**/feature/fix/docs/ branch** (right side)) and not your main branch.
- [X] Ensure that the pull request title represents the desired changelog entry.
- [X] I tested my changes and approved their functionality.
- [X] I ensured my changes do not break other parts of the code
- [X] I read and followed the [contribution guidelines](https://github.com/IntellectualSites/.github/blob/main/CONTRIBUTING.md)

## Maintainer checklist

Before the changes are marked as `Ready for merge`: 

- [ ] There are at least 2 approvals from project developers or maintainers for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title are correct
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional modification steps from users, e.g. for permission changes, make sure the entry is added to the pull request description so it can be appended to the changelog.

### CODE REVIEW

**Comment by dordsor21** [comment]
Nice idea. It might be worth doing it slightly differently however, and using the plot area name rather than world name, in "id" format (`area;X;Z`) e.g. `plotworld;-2;-4`

**Comment by Xaver106** [comment]
changed the default message to the ID format
However I don't know what you mean with using the area name? 

New default message:
image

File: Core/src/main/java/com/plotsquared/core/listener/PlotListener.java [hunk 1]
```diff
@@ -200,10 +200,11 @@ public boolean plotEntry(final PlotPlayer<?> player, final Plot plot) {
                             Caption caption = TranslatableCaption.of("notification.notify_enter");
                             Template playerTemplate = Template.of("player", player.getName());
                             Template plotTemplate = Template.of("plot", plot.getId().toString());
+                            Template worldTemplate = Template.of("world", plot.getWorldName());
                             if (!Settings.Chat.NOTIFICATION_AS_ACTIONBAR) {
-                                owner.sendMessage(caption, playerTemplate, plotTemplate);
+                                owner.sendMessage(caption, playerTemplate, plotTemplate, worldTemplate);
                             } else {
-                                owner.sendActionBar(caption, playerTemplate, plotTemplate);
+                                owner.sendActionBar(caption, playerTemplate, plotTemplate, worldTemplate);
```
Comment (NotMyFault) [inline Core/src/main/java/com/plotsquared/core/listener/PlotListener.java:207]: That could be moved into a private constructor, we're basically reusing the entire block at #plotExit below.

**Review by NotMyFault — APPROVED** [review]
Looks good so far, thanks for your contribution 👍🏻

**Comment by dordsor21** [comment]
`plot.getArea().toString()`. Provides the worldname of if the area is a partial area/one of multiple in a world, `worldname;ID`

**Review by dordsor21 — APPROVED** [review]

**Comment by NotMyFault** [comment]
This PR is ready to be merged, we will merge it in ~24 hours if there is no negative feedback, as documented by the [merge process](https://github.com/IntellectualSites/IntellectualSites/blob/main/docs/MAINTAINERS_CONTRIBUTORS.adoc#merge-process) documentation.

**Pull request merged**